frontend/src/components/utils/ButtonRadioGroup.tsx

```diff
@@ -48,6 +48,7 @@ export type ButtonRadioGroupProps = {
   options: string[];
   onChange: (newState: string) => void;
   dependentValue?: string;
+  imperativeValue?: string;
```

Comment:
confused by variable name choice: what does it mean for a value to be imperative vs dependent?